src/binary/cursor.rs

```diff
@@ -333,6 +333,10 @@ impl<R: IonDataSource> Cursor for BinaryIonCursor<R> {
         self.cursor.value.header.ion_type
     }
 
+    fn is_null(&self) -> bool {
+        self.cursor.value.is_null
+    }
+
```

Comment:
In this file: 
* Exposing an `is_null` method for use in the user-level `Reader`
* Adding support for reading timestamps with fractional seconds. For interop with the `chrono` crate, fractional seconds are turned into `nanoseconds` on the resulting `DateTime` value.